content/javascript/concepts/callbacks/callbacks.md

```diff
@@ -15,20 +15,29 @@ Callback functions are functions that are passed as arguments in other functions
 
 In JavaScript, functions can be passed as arguments because functions are objects.
 
-Suppose there are two functions, `functionA()` and `functionB()`:
+Suppose we want to create a function which makes a calculation and doubles the results of that calculation:
```

Comment:
```suggestion
Suppose a function is created which makes a calculation and doubles the results of that calculation:
```

We tend to avoid using words like `we` `you` `your` etc in the docs.